index.html

```diff
@@ -263,6 +263,18 @@ <h3 class="about_page_subtitle">Skills</h3>
               </header>
             </div>
           </div>
+      </section>    
+      <section class="contact-form">
+        <h2 class="title-form">Contact me</h2>
```

Comment:
- [ ] Ouch! this is close but as mentioned previously please make sure the size of the heading looks similar to the Figma design as I noticed yours still seems a bit smaller. So first add a selector for the class `title-form` in your styles and add a font size of around `26px` or adjust it accordingly so it looks more like the original design.
- Your design:
  ![image](https://github.com/montsecruz9/personal-portfolio/assets/104968236/b676b474-fec1-427a-80d2-97b404cc2c55)
- Figma design:
  ![image](https://github.com/montsecruz9/personal-portfolio/assets/104968236/67b69a98-fe56-46eb-ab5b-1f6f64d77caf)